docs/reference/overview_of_strategies.rst

```diff
@@ -101,7 +101,7 @@ repository.
    :header: "Original Code", "Author", "Axelrod Library Name"
 
    "GRASR_", "Unknown", "Not Implemented"
-   "K31R_", "Gail Grisell", "Not Implemented"
+   "K31R_", "Gail Grisell", ":class:`HardGoByMajority <axelrod.strategies.gobymajority.HardGoByMajority>`"
```

Comment:
Apologies: this should actually be `GoByMajority`

Reply:
Ah, I missed that when I made the changes in 708ee0bc0d961d66d0267081356f5ca198a60431.  I'll change it now.